src/app/core/classes/system-profiler.ts

```diff
@@ -113,10 +114,13 @@ export class SystemProfiler {
   }
 
   private parseDiskData(disks){
+    // Clean the slate before we start
+    this.profile.forEach(enc => enc.disks = []);
+
     let data = disks; // DEBUG
     data.forEach((item, index) => {
 
-      if(!item.enclosure){return} 
+      if(!item.enclosure){return} // Ignore boot disks
```

Comment:
`if(!item.enclosure){return}` on 123 line and `if(!enclosure){return}` on 126 line looks similar, could they be combined into single expression?